CITATION.cff

```diff
@@ -0,0 +1,25 @@
+cff-version: 1.1.0
+message: If you use this software, please cite it as below.
+authors:
+  - family-names: Sterling
```

Comment:
if we are doing quotes for the yaml, let’s be uniform. i don’t have any preference which way or the other.

Reply:
I recommend quoting all values because who knows what would happen. To be pedantic, you can also quote all the keys.